academy/4-my-own-chain/cosmjs-gui.md

```diff
@@ -191,7 +191,7 @@ Note:
 
 <HighlightBox type="info">
 
-Could an AI player and the blockchain mix together? As long as _the AI_ has access to a private key that lets it send transactions, it would be a bona fide player. The way this could be implemented is with backend scripts running on a server. See [Checkers backend scripts](./server-side.md) for an example of backend scripts with a different purpose.
+Could an AI player and the blockchain mix together? If _the AI_ has access to a private key that lets it send transactions, it would be a bona fide player. This could be implemented with backend scripts running on a server. See [Checkers backend scripts](./server-side.md) for an example of backend scripts with a different purpose.
```

Comment:
```suggestion
Could an AI player and the blockchain mix together? If _the AI_ has access to a private key that lets it send transactions, it would be a bona fide player. This could be implemented with backend scripts running on a server. See [Checkers backend scripts](./server-side.md) for an example of backend scripts for a different use-case.
```
@AndrewB9lab

Reply:
Is good.